addon/components/frost-list-pagination.js

```diff
@@ -17,23 +17,27 @@
  *   )
  * }}
  */
-
+import Ember from 'ember'
 import computed, {readOnly} from 'ember-computed-decorators'
 import {Component} from 'ember-frost-core'
 import {PropTypes} from 'ember-prop-types'
 
 import layout from '../templates/components/frost-list-pagination'
 
+const {run} = Ember
+
 export default Component.extend({
 
   // == Dependencies ==========================================================
 
   layout,
 
   // == Properties ============================================================
-
+  _pageClickCounter: 0,
```

Comment:
Never used anywhere.  See https://github.com/ciena-frost/ember-frost-list/pull/178/files#r168236617 and the either refactor or remove.